.github/workflows/build.yml

```diff
@@ -40,3 +44,12 @@ jobs:
 
     - name: Test-CC-Samples
       run: BUILD_MODE=${{ matrix.build_type }} make run-cc-tests
+
+    - name: Build-and-Run-C-Samples-with-LOC
+      run: BUILD_MODE=${{ matrix.build_type }} make clean && BUILD_MODE=${{ matrix.build_type }} CC=gcc LD=g++ L3_LOC_ENABLED=1 make run-c-tests
+
+    - name: Build-and-Run-Cpp-Samples-with-LOC
+      run: BUILD_MODE=${{ matrix.build_type }} make clean-l3 && BUILD_MODE=${{ matrix.build_type }} CC=g++ CXX=g++ LD=g++ L3_LOC_ENABLED=1 make run-cpp-tests
+
+    - name: Build-and-Run-CC-Samples-with-LOC
+      run: BUILD_MODE=${{ matrix.build_type }} make clean-l3 && BUILD_MODE=${{ matrix.build_type }} CC=g++ CXX=g++ LD=g++ L3_LOC_ENABLED=1 make run-cc-tests
```

Comment:
Sorry if this is a dumb question: what's the difference between CC and CPP?

Reply:
Well ... I was just trying to be careful to support known possible extensions for C++ file names. I've seen `*.cpp`, and `*.cc`.  So `CPP` refers to files ending with `.cpp` and `CC` refers to files ending with `.cc`.

There is also `.cXX`, but did not support that, yet, till I find a code-base that uses that too.